plugins/src/main/kotlin/com/freeletics/gradle/plugin/FreeleticsAndroidExtension.kt

```diff
@@ -10,10 +10,16 @@ import org.gradle.api.Project
 
 public abstract class FreeleticsAndroidExtension(private val project: Project) {
 
-    public fun useRoom() {
+    public fun useRoom(schemaLocation: String? = null) {
+        val processingArguments = buildList {
+            add("room.generateKotlin" to "true")
+            schemaLocation?.let {
+                add("room.schemaLocation" to it)
```

Comment:
If we just add it like this we'll have build cache relocatibility issues, we need the `RoomSchemaArgProvider`